modules/build/src/main/scala/scala/build/internal/StableScalaVersion.scala

```diff
@@ -0,0 +1,9 @@
+package scala.build.internal
+
+import upickle.default.{ReadWriter, macroRW}
+
+case class StableScalaVersion(scalaCliVersion: String, supportedScalaVersions: Seq[String])
+
+case object StableScalaVersion {
```

Comment:
No need to make it a `case object` if the `object` itself doesn't represent some data.
```suggestion
object StableScalaVersion {
```